### DISCUSSION TITLE
Is charging people for pull requests even allowed?

### DISCUSSION THREAD

**Discussion by 1404er** [community/community#12261] [Pull Requests] [answered]
My question is about [this](https://github.com/trustwallet/assets) Trust Wallet is charging people a "processing fee" to have their pull request accepted. They're charging 1000 TWT ($860 USD) and even if you've paid they can still close it for inactivity.

**Comment by Ahsoka** [comment, marked as answer]
This seems like some type of way to promote your crypto or add it to the exchange for a fee.  I don't see why this wouldn't be allowed.  It's not like they are paying for the privilege to work on code.

**Reply by 1404er** [reply]
hm ok. Just seemed odd at first

**Reply by abdolla4** [reply]
I'm a beginner and I don't know much about how to use it. Please forgive me, but my Bitcoin and ERC20 are locked. Please help me unlock them. Thank you.

**Reply by Ibnujlnsyh** [reply]
0x0000000071727De22E5E9d8BAf0edAc6f37da032
